packages/blockchain/src/db/operation.ts

```diff
@@ -14,6 +14,8 @@ import {
 
 import type { CacheMap } from './manager.js'
 
+export type OptimisticOpts = { fcUed: boolean; linked?: boolean }
+
```

Comment:
These options are not really "understandable" in a blockchain context respectively bring in concepts from the client (FcU), and a blockchain "should not need to know what an FcU is". 😛 

Can we rename this more to the point, so with what this is doing here from the blockchain perspective (I honestly also cannot read this out of the name and bring the two things together)?

Also `linked` I also can't directly read what is meant here.

These options should - if we keep - also move to `types.ts` since they become part of the official API eventually.

Reply:
(maybe then directly give this some code docs)